handler/pagination.go

```diff
@@ -0,0 +1,28 @@
+package handler
+
+import (
+	"github.com/ONSdigital/dp-population-types-api/contract"
+)
+
+func manualPagination(limit, offset int, populationTypes []string) (*contract.PopulationTypes, int, int) {
+
```

Comment:
You should attach this function to the request object and you can use the limit and offset directly

Reply:
sorry not sure what you mean by this, could you elaborate a litter furhter? apologies

Reply:
use GetPopulationTypesResponse as a receiver like:
```
resp := GetPopulationTypesResponse{limit: req.Limit, offset: req.Offset}
func (r *GetPopulationTypesResponse) paginate(populationTypes []string)...{
  // do stuff with r.Limit, r.Offset and r.PopulationTypes
}
```
Something like that